WordPress/src/main/java/org/wordpress/android/datasets/SiteSettingsTable.java

```diff
@@ -29,6 +30,7 @@
     public static void createTable(SQLiteDatabase db) {
         if (db != null) {
             db.execSQL(SiteSettingsModel.CREATE_SETTINGS_TABLE_SQL);
+            db.execSQL(JetpackSettingsModel.CREATE_JP_SETTINGS_TABLE_SQL);
```

Comment:
I am probably missing something here. Since it's a new model and separate from the Site Settings, why is it harder to do in FluxC (aside from a couple dispatch calls)?